src/snap/flags.cc

```diff
@@ -0,0 +1,395 @@
+#include "flags.h"
+//#include <snap/flags.h>
+
+#include <map>
+#include <iostream>
+#include <boost/any.hpp>
+#include <spdlog/spdlog.h>
+
+// Set default option values
+#define STAND_ALONE_PORT 8181
```

Comment:
It seems that `STAND_ALONE_PORT` should be changed to 8182
for plugin written in golang I see following help:
```
--stand-alone-port value      specify http port when stand-alone is set (default: 8182)

```